client/component/CheckoutForm.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import styled from 'styled-components';
 import style from 'styled-components';
```

Comment:
a little confused why youre using 2 different imports for styled components, IF style is a property you need you can do 
import { style, styled } from 'styled-components';

Reply:
Yeah that was a typo. I have fixed that in my current iteration of my code